models/reservation.js

```diff
@@ -152,6 +98,21 @@ exports.finish = function(reservationId) {
     });
 };
 
+exports.review = function(reservationId, rating, comment) {
+  var reservation;
+  return exports.updateStatus(reservationId, exports.ReservationStatusEnum.reviewed)
+    .then(function(res) {
+      reservation = res;
+      return Promise.all([
+        Spot.review(reservation.attributes.spotId, reservationId, rating, comment),
+        User.removeReservationFromDriverActive(reservation)
+      ]);
+    })
+    .then(function() {
+      return reservation;
```

Comment:
ya I realized that in the Promise.all we don't actually change the reservation, so it totally doesn't matter if we fetch a new one. Though we probably should for consistency, and since FB does solid caching I shouldn't worry about the network call.